lib/alchemy/upgrader/four_point_zero.rb

```diff
@@ -58,8 +159,16 @@ def alchemy_4_0_todos
 The code for the available contents button in the element editor partial can be removed
 without replacement. The nested elements editor partials render automatically.
 
+We have an experimental automatic upgrader which you can call using
```

Comment:
Isn't the upgrade task invoked automatically while running `rake alchemy:upgrade`? So this note seems redundant. We should better print further instructions in the conversion task.